client/stylesheets/_modal.scss

```diff
@@ -1034,6 +1034,34 @@ div[class="sp-replacer sp-light full-spectrum sp-active"] {
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
 
 
+/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+	Join Modal
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
+
+/*~~~~ Join Button ~~~~*/
+
+.modJoinTrigger {
+     color: #fff !important;
+     border: 1px solid #fff !important;
+     padding: 10px 14px;
+     background-color: #f8504b !important;
+     font-size: 0.9rem !important;
+}
+
+.modJoinTrigger:hover {
+     color: #404040 !important;
+     background-color: #fff !important;
+}
+
+/*~~ Join Button End ~~*/
+
+.modJoinWrapper {
```

Comment:
Empty rule<br>Selector `modJoinWrapper` should be written in lowercase with hyphens